base/api-deployment.yaml

```diff
@@ -204,6 +204,8 @@ spec:
               value: 'https://api.getbase.com'
             - name: ZENDESK_SELL_API_KEY
               value: '$(ZENDESK_SELL_API_KEY)'
+            - name: FF_NOTIFICATION_CELERY_PERSISTENCE
+              value: '$(FF_NOTIFICATION_CELERY_PERSISTENCE)'
```

Comment:
Damn, I did noticed that this entry wasn't in there when I worked on restricting the API for RCMP. I thought you figured something out and that was not necessary.

Reply:
And in fact, I think it might not be the only one missing IIRC. We should double-check for other entries.